src/elements/Chip/ChipTheme.ts

```diff
@@ -102,69 +138,146 @@ export const chipTheme: ChipTheme = {
   sizes: baseTheme.sizes,
   focus: baseTheme.focus,
   variants: {
-    filled: 'text-white',
-    outline: 'bg-opacity-0 border hover:bg-transparent text-surface-content',
-    text: 'bg-opacity-0 border-0'
+    filled: 'border border-transparent box-border',
+    outline: 'bg-opacity-0 border hover:bg-transparent'
   },
   colors: {
     default: {
-      base: 'bg-gray-600 border-gray-600 text-gray-300',
-      selectable: [
-        baseTheme.colors.default.selectable,
-        'hover:bg-gray-500 hover:border-gray-500'
-      ].join(' '),
-      selected: '!bg-gray-500 text-white'
+      variants: {
+        filled: 'bg-panel border-panel text-panel-content',
+        outline: 'text-panel-content border-panel-content'
+      },
+      selectable: {
```

Comment:
Interesting that selectable changes it this much.

Reply:
ye, adds a bunch of hover state styling that's different for the two variants

Reply:
Is there anyway you could use the lighten or darken to achieve this easier? Just an idea

Reply:
Considering sometimes the background color, text color, and border entirely change, I don't think I can simplify it to lightening or darkening 🥲